semantics_analysis/relation_extractor/ontology_utils.py

```diff
@@ -859,9 +859,9 @@
                 'description': 'метрика не используется для указанной задачи',
                 'examples': [
                     {
-                        'text': 'Мы не можем использовать MSE для оценки классификации.',
-                        'Metric': 'MSE',
-                        'Task': 'классификации'
+                        'text': 'Для задач бинарной классификации я измеряю ROC AUC, а в задачах многоклассовой классификации – точность (accuracy).',
+                        'Metric': 'ROC AUC',
+                        'Task': 'задачах многоклассовой классификации'

```

Comment:
I think this example can be confusing for our model as `ROC AUC` metric is actually used for the classification task with multiple classes.

The ROC curve is built for every class independently.

I understand that this is a "context based" example, but perhaps we can improve it.

Reply:
It's a good comment cause this problem needs some discussion.
You're absolutely right about RC AUC, but we use external knowledge.
In my opinion we should base relations on the context due to precision. If person used some metric and didn't use another we should extract only first relation.
If you have any suggestions for improvement, I'm all ears :)

Reply:
Perhaps this example can be useful for the model: we show that the "context-based" knowledge is more valuable and important rather than the external knowledge.

So let's keep this example for now.